tests/plugins/test_cluster_integration.py

```diff
@@ -55,7 +55,8 @@ def environment_combinations():
 
 
 @pytest.mark.parametrize(
-    "plugin_cls", [DDPPlugin, DDPShardedPlugin, DDP2Plugin, pytest.param(DeepSpeedPlugin, marks=RunIf(deepspeed=True))]
+    "plugin_cls",
+    [DDPPlugin, DDPShardedPlugin, DDP2Plugin, pytest.param(DeepSpeedStrategy, marks=RunIf(deepspeed=True))],
 )
 def test_ranks_available_manual_plugin_selection(plugin_cls):
```

Comment:
maybe need to update the parameter name and test name here ?
plugin -> strategy

Reply:
That would be too far reaching and should be done for follow up PRs. See [#10548 ](https://github.com/PyTorchLightning/pytorch-lightning/issues/10549#issuecomment-997863968)